core/src/main/java/dev/hypera/scaffolding/instance/SchematicChunkLoader.java

```diff
@@ -1,34 +1,7 @@
-/*
```

Comment:
This copyright header should **not** be removed.

Reply:
I had problems with utf-8 compiling, so i removed it, i dont know what you did, or wrote in those headers, but on 3 of my windows machines, the whole project was corrupted.